src/main/java/storage/Storage.java

```diff
@@ -0,0 +1,97 @@
+package storage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Scanner;
+
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.TaskList;
+import task.ToDo;
+
+import ui.Ui;
+
+
+/**
+ * The {@code Storage} class handles the loading and saving of tasks to and from a file.
+ * It interacts with the {@code TaskList} and {@code Ui} classes to manage the tasks and provide feedback to the user.
+ */
+public class Storage {
+
+    // The file path where tasks are saved and loaded from
+    static final String FILE_PATH = "data/bob.txt";
+    private Ui ui = new Ui();
+
+    /**
+     * Loads tasks from the specified file into the given {@code TaskList}.
+     *
+     * @param tl The {@code TaskList} where tasks will be loaded into.
+     */
+    public void loadTasks(TaskList tl) {
+        File f = new File(FILE_PATH);
+        try {
+            Scanner s1 = new Scanner(f);
+            while (s1.hasNext()) {
+                String t = s1.nextLine();
+                String[] task_list = t.trim().split(" \\| ");
+                Task x;
```

Comment:
I like the general consistency of your abbreviations. This particular Task object is abbreviated as x instead of t like in other places though. Any reason why the String object is abbreviated as t and the Task object is abbreviated as x, rather than abbreviating the task object as t and using another abbreviation for the String object?